### ISSUE TITLE
Improve confusion matrix and score distributions

### ISSUE THREAD

**Issue by desilinguist** [EducationalTestingService/rsmtool#151]
Right now, the confusion matrix and the score distribution plots rely entirely on the test set human scores. This means that if there are, say, 6s  in the training data but none in the test data, the confusion matrix will only be 5x5 and the score distribution plot will only show 5 bars. Perhaps it would be better to take the union of the human scores from both the training and the test sets?
